velox/functions/sparksql/aggregates/CentralMomentsAggregate.cpp

```diff
@@ -60,6 +71,115 @@ exec::AggregateRegistrationResult registerCentralMoments(
             .argumentType(inputType)
             .build());
   }
+  return signatures;
+}
+
+exec::AggregateRegistrationResult registerSkewness(
+    const std::string& name,
+    bool withCompanionFunctions,
+    bool overwrite) {
+  std::vector<std::shared_ptr<exec::AggregateFunctionSignature>> signatures =
+      getSignatures();
+
+  return exec::registerAggregateFunction(
+      name,
+      std::move(signatures),
+      [name](
+          core::AggregationNode::Step step,
+          const std::vector<TypePtr>& argTypes,
+          const TypePtr& resultType,
+          const core::QueryConfig& config) -> std::unique_ptr<exec::Aggregate> {
+        VELOX_CHECK_LE(
```

Comment:
I think this function should require exactly one argument, rather than allowing at most one.